### PR TITLE
netex export to geojson, netex zip dowload service

### PR DESCRIPTION
**Pull request by philosotractor**
# Added
* external interface 'data-content' to netex coversion
* nap-generated netex package links to service geojson export
* export/netex endpoint for downloading netex zip
*  db: add specql for netex-conversion table's enum interface-data-content
   db: netex-conversion add column 'data-content'

### CODE REVIEW

File: ote/src/clj/ote/integration/export/geojson.clj [hunk 1]
```diff
@@ -59,41 +60,74 @@
...
+        service (when operator
+                  (-> (first
+                        (specql/fetch db
+                                      ::t-service/transport-service
+                                      transport-service-properties-columns
+                                      {::t-service/transport-operator-id transport-operator-id
+                                       ::t-service/id transport-service-id
+                                       ::t-service/published op/not-null?}))
+                      (append-nap-generated-netex-file-links db config transport-service-id)
+                      (link-to-companies-csv-url)
```
Comment (rckopteri) [inline ote/src/clj/ote/integration/export/geojson.clj]: remove this

File: ote/src/clj/ote/integration/export/geojson.clj [hunk 2]
```diff
@@ -59,41 +60,74 @@
...
+  (let [areas (seq
+                (fetch-operation-area-for-service db
+                                                  {:transport-service-id transport-service-id}))
+        operator (when areas
+                   (-> (first
+                         (specql/fetch db
+                                       ::t-operator/transport-operator
+                                       transport-operator-properties-columns
+                                       {::t-operator/id transport-operator-id}))
+                       ;; Personal data should not be exported due to privacy requirement
```
Comment (rckopteri) [inline ote/src/clj/ote/integration/export/geojson.clj]: Not personal data. It is company data
So the comment could be something like
;; Due to privacy requirement remove gsm, phone, address and email from operator export

File: ote/src/clj/ote/integration/import/gtfs.clj [hunk 3]
```diff
@@ -301,27 +301,31 @@
 (defn download-and-store-transit-package
```
Comment (rckopteri) [inline ote/src/clj/ote/integration/import/gtfs.clj]: This receives quite many parameters and I was wondering if this fn does more than it should

Comment (philosotractor) [reply ote/src/clj/ote/integration/import/gtfs.clj]: Agree, but that's something for a new ticket :)

**Pull request merged**